platform/windows/display_server_windows.cpp

```diff
@@ -4893,6 +4893,7 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam) {
 void DisplayServerWindows::_process_activate_event(WindowID p_window_id) {
 	WindowData &wd = windows[p_window_id];
 	if (wd.activate_state == WA_ACTIVE || wd.activate_state == WA_CLICKACTIVE) {
+		last_button_state = 0;
```

Comment:
Could this break drag&drop operations across applications? e.g. minimizing a game window, starting a file drag, hovering the mouse over the game window's taskbar icon, then releasing the mouse.

Or after #67531 is merged, drag&drop of Nodes between multiple windows of one game app, switching between them using alt+tab or hovering taskbar icon.

Also the button is released both on focus gain and loss for Windows, while only focus loss for the other OS.

Reply:
Should not break file dnd, but definitely will break 67531, so I guess getting current state instead of saving way (similar to https://github.com/godotengine/godot/pull/92415) might be better option.